lib/pf/CHI.pm

```diff
@@ -62,7 +62,7 @@ Hash::Merge::specify_behavior(
     'PF_CHI_MERGE'
 );
 
-our @CACHE_NAMESPACES = qw(configfilesdata configfiles httpd.admin httpd.portal pfdns switch.overlay ldap_auth omapi fingerbank);
+our @CACHE_NAMESPACES = qw(configfilesdata configfiles httpd.admin httpd.portal pfdns switch.overlay ldap_auth omapi fingerbank t);
```

Comment:
No need to add a test namespace.
Add the following in the t/data/chi.conf

[namespace t]
driver=File
root_dir=/tmp/chi